packages/lesswrong/components/posts/PostsPage/PostsPageWrapper.tsx

```diff
@@ -3,33 +3,63 @@ import { Components, registerComponent } from '../../../lib/vulcan-lib';
 import { useSingle, UseSingleProps } from '../../../lib/crud/withSingle';
 import { isMissingDocumentError, isOperationNotAllowedError } from '../../../lib/utils/errorUtil';
 import { isPostWithForeignId } from "./PostsPageCrosspostWrapper";
+import { commentGetDefaultView } from '../../../lib/collections/comments/helpers';
+import { useCurrentUser } from '../../common/withUser';
+import { useMulti } from '../../../lib/crud/withMulti';
+import { viewNames } from '../../comments/CommentsViews';
+import { useSubscribedLocation } from '../../../lib/routeUtil';
 
 const PostsPageWrapper = ({ sequenceId, version, documentId }: {
   sequenceId: string|null,
   version?: string,
   documentId: string,
 }) => {
+  const currentUser = useCurrentUser();
+  const { query } = useSubscribedLocation();
+
+  const extraVariables = {sequenceId: 'String', ...(version && {version: 'String'}) }
```

Comment:
Have you checked that version still works? (I'm not actually sure off the top of my head how version gets used, and I'm an increasingly lazy reviewer so I'm asking you.)

Reply:
I have now checked (but I hadn't before)